camera_calibration/nodes/cameracalibrator.py

```diff
@@ -56,6 +57,17 @@ def optionsValidCharuco(options, parser):
     # TODO: check for fisheye and stereo (not implemented with ChArUco)
     return True
 
+def compute_robustness_treshold(size: List[int], old_robustness_treshold: int = None):
```

Comment:
If you are adding typing, please specify the return value

Reply:
in the end I haven't used the function and therefore I've removed typing and the function itself